src/main/resources/templates/govuk/radio.ftl

```diff
@@ -101,7 +101,7 @@
   </#if>
 </#macro>
 
-<#macro radioGroup path nestingPath="" label="" hiddenContent=true legendSize="h1" showNestedForInternetLabels=false>
+<#macro radioGroup path nestingPath="" label="" hiddenContent=true legendSize="h1" showNestedForInternetLabels=true>
```

Comment:
It looks like when selecting an option that doesn't have any content (e.g. pre 2021 for `/categories/dishwashers?mode=INTERNET`) it still adds an empty `govuk-radios__conditional` which has some padding and causes the second radio to get pushed down a bit. Would that be easy to fix?